Content.Server/GameObjects/Components/Mobs/Body/Organs/MiscOrgan.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace Content.Server.GameObjects.Components.Mobs.Body.Organs
+{
+    public class Misc : Organ { }
```

Comment:
Just remove this if it's not gonna do anything.

If you have concrete plans for it, make a comment here explaining its purpose.

Reply:
it's placeholder organs used for gibbing now